hail/src/main/scala/is/hail/expr/ir/EmitClassBuilder.scala

```diff
@@ -322,6 +322,16 @@ class EmitClassBuilder[C](
           cb.assign(ms, false)
           cb.assign(vs, value)
         })
+
+    override def get(cb: EmitCodeBuilder): PValue = {
+      if (_pt.required) {
+        vs
+      } else {
+        // TODO: I don't think this is right. Seems to get an open ended issue instead of a fatal error.
+        cb.ifx(ms, cb._fatal(s"Can't convert missing ${_pt} to PValue"))
```

Comment:
At some point I got an open ended error from this code, but after some changes I think I stopped hitting this code path. It may be that the open ended error I briefly saw from here was a result of some other issue somewhere else, and that it was resolved. Just wanted to flag this with a TODO to see if you thought it looked right.